python_modules/dagster/dagster/core/definitions/config.py

```diff
@@ -18,6 +18,7 @@ class ConfigMapping(
         [
             ("config_fn", Callable[[Any], Any]),
             ("config_schema", IDefinitionConfigSchema),
+            ("enable_pre_processing", bool),
```

Comment:
don't love this name, maybe "receive_processed_config_values"

definitely deserves a clarifying comment